client/lib/upgrades/actions/purchases.js

```diff
@@ -66,6 +68,10 @@ function clearPurchases() {
 	} );
 }
 
+function clearSitePlans( siteId ) {
+	Dispatcher.handleViewAction( clearSitePlansAction( siteId ) );
```

Comment:
I think we have to use `context.store.dispatch` instead of this, but I don't think this can be done from a library which does not have access to the global redux store.